task/02-numbers-tasks.js

```diff
@@ -73,7 +75,7 @@ function getAverage(value1, value2) {
  *   (-5,0) (10,-10) => 18.027756377319946
  */
 function getDistanceBetweenPoints(x1, y1, x2, y2) {
-    throw new Error('Not implemented');
+   return Math.sqrt(Math.pow((x2-x1),2)+Math.pow((y2-y1),2));
```

Comment:
Please, improve code using Math.hypot

Please, try to follow these rules https://javascript.info/coding-style